src/constants/indicators.js

```diff
@@ -221,37 +221,37 @@ export default {
   },
   unemploymentRateGeneral: {
     name: 'Unemployment rate',
-    format: '0[.]0%',
+    format: '0[.]0[0]%',
     contextual: false,
     lastUpdated: '2017',
     source: 'Unemployment rate, Persons, % of population by sex',
   },
   unemploymentRateIndigenous: {
     name: 'Indigenous unemployment rate',
     legendText: 'Indigenous',
-    format: '0[.]0%',
+    format: '0[.]0[0]%',
```

Comment:
The way you've done these formats looks like a sensible heuristic. 

There's lots of repetition here, though. I reckon they should go in a constants block at the top of the file e.g.

```
FORMATS = {
  PERCENT_CONCISE: '0[.]0[0]%',
// ... etc
```

Reply:
Good idea. Fixed.